cmd/gossamer/config.go

```diff
@@ -444,13 +467,49 @@ func setDotGlobalConfig(ctx *cli.Context, tomlCfg *ctoml.Config, cfg *dot.Global
 	}
 
 	cfg.NoTelemetry = ctx.Bool("no-telemetry")
+}
 
-	logger.Debug(
-		"global configuration",
-		"name", cfg.Name,
-		"id", cfg.ID,
-		"basepath", cfg.BasePath,
-	)
+func setDotGlobalConfigName(ctx *cli.Context, tomlCfg *ctoml.Config, cfg *dot.GlobalConfig) error {
+	globalBasePath := utils.ExpandDir(cfg.BasePath)
+	initialised := dot.NodeInitialized(globalBasePath, false)
+
+	thereIsNameTomlConfig := tomlCfg.Global.Name != ""
```

Comment:
No need to create these variables `thereIsNameTomlConfig` and `thereIsNameFlag`
Directly check the condition in `if-else`